server/apps/postgres-app/sample-configuration-distributed/opensearch.properties

```diff
@@ -94,7 +94,7 @@ opensearch.indexAttachments=false
 #
 # Please note that custom overrides can be defined here.
 #
-# opensearch.search.overrides=org.apache.james.mailbox.postgres.search.AllSearchOverride,org.apache.james.mailbox.postgres.search.DeletedSearchOverride, org.apache.james.mailbox.postgres.search.DeletedWithRangeSearchOverride,org.apache.james.mailbox.postgres.search.NotDeletedWithRangeSearchOverride,org.apache.james.mailbox.postgres.search.UidSearchOverride,org.apache.james.mailbox.postgres.search.UnseenSearchOverride
+opensearch.search.overrides=org.apache.james.mailbox.postgres.search.AllSearchOverride,org.apache.james.mailbox.postgres.search.DeletedSearchOverride, org.apache.james.mailbox.postgres.search.DeletedWithRangeSearchOverride,org.apache.james.mailbox.postgres.search.NotDeletedWithRangeSearchOverride,org.apache.james.mailbox.postgres.search.UidSearchOverride,org.apache.james.mailbox.postgres.search.UnseenSearchOverride
```

Comment:
Not enabled by default I think, pretty much what we do with cassandra

Reply:
I think it will have better performance, and it should be enabled by default.

Reply:
Not proved yet by perf tests :) Facts first !